app/views/responses/show.html.erb

```diff
@@ -35,19 +35,18 @@
 
 <!-- Button trigger modal -->
 
-
 <!-- Modal -->
 <div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
   <div class="modal-dialog modal-dialog-centered" role="document">
     <div class="modal-content">
       <div class="modal-header">
-        <h5 class="modal-title" id="exampleModalCenterTitle">Generate Results</h5>
+        <h5 class="modal-title" id="exampleModalCenterTitle">Show me where to eat</h5>
         <button type="button" class="close" data-dismiss="modal" aria-label="Close">
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
       <div class="modal-body">
-        Are you sure that you want to end the poll prematurely and generate your restaurant result?
+        Are you sure that you want to end the poll prematurely and see your restaurant result?
```

Comment:
End the poll now and show me where to eat!

Reply:
> Button can be Yes / No (to cancel)

👌

Reply:
> End the poll now and show me where to eat!
![localhost_3000_chomp_sessions_944532_responses_3(iPhone 6_7_8)](https://user-images.githubusercontent.com/11084577/135367930-5135abb6-5829-42a6-8fb3-2b4dd6fdce60.png)

This ok?